test/runtests.jl

```diff
@@ -14,4 +14,5 @@ println("Testing SuiteSparseGraphBLAS.jl")
 @testset "SuiteSparseGraphBLAS" begin
     include_test("gbarray.jl")
     include_test("operations.jl")
+    include_test("testrules.jl")
```

Comment:
Usually the structure of `test` folder mirrors the `src/` folder, which makes it easier to find things when the package grows.